### PR TITLE
chore(tests): remove some useless code

### PR DESCRIPTION
**Pull request by sileht**
This reduces the time needed to record tests.

Change-Id: I2de9aaa8396b9f7138f85dc0178990fa3ec6fd9d

### CODE REVIEW

File: mergify_engine/tests/functional/actions/test_squash.py [hunk 1]
```diff
@@ -34,23 +34,18 @@ async def test_squash_several_commits_ok(self):
...
+            "checkout", "--quiet", f"origin/{self.main_branch_name}", "-b", branch_name
         )
 
         for i in range(0, 3):
             open(self.git.tmp + f"/file{i}", "wb").close()
             await self.git("add", f"file{i}")
             await self.git("commit", "--no-edit", "-m", f"feat(): add file{i+1}")
 
-        await self.git("push", "--quiet", repo_name, branch_name)
+        await self.git("push", "--quiet", "fork", branch_name)
```
Comment (jd) [inline mergify_engine/tests/functional/actions/test_squash.py:48]: This seems unrelated at first sight?

Comment (sileht) [reply mergify_engine/tests/functional/actions/test_squash.py:48]: some tests what using the main branch of the fork instead of the main branch of the origin repository.
Since I don't create the main anymore on the fork, I have to change all tests using the main branch of the fork.

**Review by jd — APPROVED** [review]
